docs/source/example_gnpe_model.md

```diff
@@ -0,0 +1,90 @@
+# GNPE model (production)
+
+This tutorial has the highest profile settings and is the one typically used for production use.
+The main difference from the [NPE](example_npe_model.md) tutorial is that here we are now using [GNPE](gnpe.md)
+(group neural posterior estimation). The data generation is exactly the same as the [previous](example_npe_model.md)
+tutorial, but we repeat it here, for completeness.
+
+
+Step 1 Generating a Waveform Dataset
+------------------------------------ 
+
+We generate the waveform dataset locally:
+
+```
+cd dingo
+mkdir $(pwd)/gnpe_model_train_dir
+export TRAIN_DIR=$(pwd)/gnpe_model_train_dir
+dingo_generate_dataset --settings examples/gnpe_model/waveform_dataset_settings.yaml --out_file $TRAIN_DIR/waveform_dataset.hdf5
+```
+
+or using condor:
+
+```
+dingo_generate_dataset_dag --settings_file
+$(pwd)/examples/gnpe_model/waveform_dataset_settings.yaml --out_file
+$TRAIN_DIR/IMRPhenomXPHM.hdf5 --env_path $DINGO_VENV_PATH --num_jobs 4
+--request_cpus 16 --request_memory 1280000 --request_memory_high 256000
+```
+
+
+Step 2 Generating an ASD dataset
+--------------------------------
+
+As before we generate a fiducial ASD dataset containing a single ASD:
+
+```
+dingo_generate_asd_dataset --settings_file examples/gnpe_model/asd_dataset_settings_fiducial.yaml --data_dir
+$TRAIN_DIR/asd_dataset_folder -out_name $TRAIN_DIR/asds_O1_fiducial.hdf5
+
+and a large ASD dataset:
+
+dingo_generate_asd_dataset --settings_file $(pwd)/examples/gnpe_model/asd_dataset_settings.yaml --data_dir
+$TRAIN_DIR/asd_dataset_folder -out_name $TRAIN_DIR/asds_O1.hdf5
+```
+
+
+Step 3 Training the network
+---------------------------
+
+Now we are ready for training using GNPE. Here we need to train two networks, one which estimates the time of arrival 
+in the detectors and one which does the full inference task. First we train the initialization network for the detector times with:
```

Comment:
There needs to be some discussion of *why* we are doing what we are doing, e.g., what GNPE is, why it needs two networks, why the initialization network is smaller, etc. It may also be helpful to visualize by including the flowchart from the quickstart tutorial.